models/person.rb

```diff
@@ -3,21 +3,20 @@
 require_relative 'trimmer_decorator'
 
 class Person < Nameable
+  # getters and setters
+  attr_reader :id, :rentals
+  attr_accessor :name, :age
+
   # Constructor
   def initialize(age, name = 'Unkown', parent_permission: true)
     super()
     @id = Random.rand(1..1000)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
```

Comment:
- [ ] It's great you initialized your `rentals`. However, I couldn't find any method for adding rentals in your Person class. Make sure you have created a month for adding rentals here 😉